src/GetOrCreateImage/GetOrCreateImage.js

```diff
@@ -46,8 +46,8 @@ const GetOrCreateImage = async event => {
 
       // Required try/catch because Sharp.catch() doesn't seem to actually catch anything. 
       try {
-        resizedImage = Sharp(imageObj.Body)
-          .resize(width, height)
+        resizedImage = Sharp(imageObj.Body, { animated: true })
+          .resize(width, height, {fit: 'inside'})
```

Comment:
Have you confirmed that `{fit: 'inside'}` doesn't have side effects for other formats?